localStorage.js

```diff
@@ -1,29 +1,25 @@
-const nameI = document.getElementById('#name');
-const emails = document.getElementById('#email');
-const message = document.getElementById('#message');
-const submit = document.getElementById('#submit');
+const nameI = document.getElementById('name');
+const emails = document.getElementById('email');
+const message = document.getElementById('message');
+const submit = document.getElementById('submit');
 
-function setLocalStorage() {
+function saveLocalStore() {
   const data = {
     name: nameI.value,
     email: emails.value,
     message: message.value,
   };
   localStorage.setItem('data', JSON.stringify(data));
 }
-submit.addEventListener('click', () => {
-  if (null) {
-    setLocalStorage();
-  }
-},
+submit.addEventListener('click', saveLocalStore);
```

Comment:
- [ ] Great job with the save and load functions, as they do work, but this event listener here does not meet the requirement screenshotted below, the data should save every time the user changes the input, to do that, you'll have to add the event listener to each one of the form fields instead while using the property 'input' instead of click.
![image](https://user-images.githubusercontent.com/73140461/146451973-f741e0bb-4371-4e1b-8305-14e117c26940.png)